libwayshot/src/lib.rs

```diff
@@ -366,64 +352,45 @@ impl WayshotConnection {
             tracing::error!("You can send a feature request for the above format to the mailing list for wayshot over at https://sr.ht/~shinyzenith/wayshot.");
             return Err(Error::NoSupportedBufferFormat);
         };
-        Ok(FrameCopy {
+        let frame_copy = FrameCopy {
             frame_format,
             frame_color_type,
             frame_mmap,
-            transform,
-        })
+            transform: output_info.transform,
+            position: capture_region
+                .map(|capture_region| {
+                    let logical_region = capture_region.logical();
+                    (logical_region.inner.x as i64, logical_region.inner.y as i64)
+                })
+                .unwrap_or_else(|| {
+                    (
+                        output_info.dimensions.x as i64,
+                        output_info.dimensions.y as i64,
+                    )
+                }),
+        };
+        tracing::debug!("Created frame copy: {:#?}", frame_copy);
+        Ok((frame_copy, frame_guard))
     }
 
-    fn create_frame_copy(
+    pub fn capture_frame_copies(
```

Comment:
since this function is public ,so I think add docs is needed

Reply:
We might be able to keep this private. I forgot if it was used, I might've forgot to undo this.